src/commands/mod.rs

```diff
@@ -11,6 +12,14 @@ pub enum Command {
     #[command(name = "nsec3-hash")]
     Nsec3Hash(self::nsec3hash::Nsec3Hash),
 
+    /// Send a NOTIFY packet to DNS servers
+    ///
+    /// This tells them that an updated zone is available at the master servers. It can perform TSIG
```

Comment:
Is this just newer terminology or does it mean anything different? In the first case, I definitely agree. But this came from the manpage of ldns-notify, so decided to copy that first.

Reply:
It's quite old terminology actually (which had been replaced by master/slave some decades ago), but is now changed to primary/seconday again, as in many other environments (git, parallel prog., etc.).
It means the same, and is known terminology, the ldns man pages are just old.

Reply:
I think our preference is to avoid using words that are known to be offensive to some and use alternate replacement terms where available. Master and slave should indeed I believe be replaced by primary and secondary. [RFC 9499](https://datatracker.ietf.org/doc/html/rfc9499) states:

> Slave server: See "Secondary server".
> Master server: See "Primary server".

Reply:
Excellent! Then I'll do that, because that terminology should indeed disappear